definitions/3.0.0/multiFormatSchema.json

```diff
@@ -1,5 +1,11 @@
 {
   "description": "The Multi Format Schema Object represents a schema definition. It differs from the Schema Object in that it supports multiple schema formats or languages (e.g., JSON Schema, Avro, etc.).",
+  "type": "object",
+  "patternProperties": {
+    "^x-[\\w\\d\\.\\x2d_]+$": {
+      "$ref": "http://asyncapi.com/definitions/3.0.0/specificationExtension.json"
+    }
+  },
   "if": {
```

Comment:
If you want, feel free to remove this if/then as https://github.com/asyncapi/spec-json-schemas/pull/507 does :v: